### PR TITLE
Add base

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr review please

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ndoh-hub
+NDOH Registration and Change service for MomConnect and NurseConnect
+
+## Registration validity requirements
+All registrations should have the following information:
+- contact (identity-store id)
```
Comment (gsvr) [inline README.md]: identity?

File: registrations/models.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    STAGE_CHOICES = (
+        ('prebirth', "Mother is pregnant"),
+        ('postbirth', "Baby has been born"),
+        ('loss', "Baby loss")
+    )
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    stage = models.CharField(max_length=30, null=False, blank=False,
+                             choices=STAGE_CHOICES)
+    mother_id = models.CharField(max_length=36, null=False, blank=False)
```
Comment (gsvr) [inline registrations/models.py]: Change to `registrant_id` since this may accept nurseconnect registrations.

File: registrations/tests.py [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+        # Setup
+        valid_id_no = "12345"
+        invalid_id_no = 12345
+        # Execute
+        # Check
+        self.assertEqual(is_valid_id_no(valid_id_no), True)
+        self.assertEqual(is_valid_id_no(invalid_id_no), False)
+
+
+# class TestRegistrationValidation(AuthenticatedAPITestCase):
```
Comment (gsvr) [inline registrations/tests.py]: Remove commented out code?

File: registrations/models.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    """ The source from which a registation originates.
+        The User foreignkey is used to identify the source based on the
+        user's api token.
+    """
+    AUTHORITY_CHOICES = (
+        ('patient', "Patient"),
+        ('advisor', "Trusted Advisor"),
+        ('hw_limited', "Health Worker Limited"),
+        ('hw_full', "Health Worker Full")
+    )
```
Comment (gsvr) [inline registrations/models.py]: Perhaps we should add something here for instances like NurseConnect and PMTCT.

**Comment by gsvr** [comment]
First round done.

File: registrations/models.py (hunk 2 above)
Comment (imsickofmaps) [inline registrations/models.py]: Agreed, go ahead

File: registrations/tests.py (hunk 3 above)
Comment (imsickofmaps) [inline registrations/tests.py]: Will be re-added in next PR, so just leave

File: registrations/models.py (hunk 4 above)
Comment (imsickofmaps) [inline registrations/models.py]: I think these are fine. The authority is the same. There will be an additional Source record in the model, but that's not a source code change.

File: README.md (hunk 1 above)
Comment (imsickofmaps) [inline README.md]: yes

**Comment by imsickofmaps** [comment]
@gsvr please make these changes and I'll review

**Comment by gsvr** [comment]
@imsickofmaps Please review.

**Comment by imsickofmaps** [comment]
@gsvr pep8 then land 👍

**Pull request merged**